README.md

```diff
@@ -20,6 +20,16 @@ The next section details the prerequisites to install Lisk Core from source usin
 
 ### System Install
 
+* Create a new user
+
+  * Ubuntu 14|16 / Debian:
```

Comment:
On https://lisk.io/documentation/lisk-core/setup/pre-install/binary we list Ubuntu 16.04 and 18.04